incubator/aci/create-k8-connector/script.md

```diff
@@ -31,11 +31,17 @@ CLI using the instructions below.  Find your subscriptionId with the
 az CLI:
 
 Use az to create a Service Principal that can perform operations on
-your resource group:
+your subscription:
 
 ```
 SUBSCRIPTION_ID=$(az account show | jq -r '.id')
-SP_JSON=$(az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$SIMDEM_RESOURCE_GROUP")
+SP_JSON=$(az ad sp create-for-rbac --role="Contributor" --scopes="/subscriptions/$SUBSCRIPTION_ID")
+```
+##Register the app
```

Comment:
Can you please explain what is happening here. What is the "app" and why do we need to register it?

Reply:
"resource provider - A service that supplies the resources you can deploy and manage through Resource Manager. Each resource provider offers operations for working with the resources that are deployed. Some common resource providers are Microsoft.Compute, which supplies the virtual machine resource, Microsoft.Storage, which supplies the storage account resource, and Microsoft.Web, which supplies resources related to web apps." So ACI is provided through Container Instance resource manager - usually this linkage is automatic but for some people it might not be

Reply:
I meant can you explain it in the doc :-) 

Also note you have a typo here (missing space between '#' and 'Register'